server/src/main/java/org/opensearch/cluster/coordination/CoordinationState.java

```diff
@@ -571,6 +577,9 @@ public void handleCommit(ApplyCommitRequest applyCommit) {
         );
 
         persistedStateRegistry.getPersistedState(PersistedStateType.LOCAL).markLastAcceptedStateAsCommitted();
+        if (isRemoteStateEnabled && localNode.isClusterManagerNode()) {
```

Comment:
Why do we need to `markLastAcceptedStateAsCommitted` for both local and remote PersistedStateType?

Reply:
It will not break anything if we don't mark RemotePersistedState as committed. But, for consistency between InMemoryPersistedState and RemotePersistedState, marking the RemotePersistedState as committed too on follower nodes as well. We were already doing this on elected cluster manager node.